mscore/file.cpp

```diff
@@ -792,9 +792,14 @@ QStringList MuseScore::getOpenScoreNames(const QString& filter, const QString& t
       {
       QSettings settings;
       QString dir = settings.value("lastOpenPath", preferences.myScoresPath).toString();
+
       if (preferences.nativeDialogs) {
-            return QFileDialog::getOpenFileNames(this,
-               title, dir, filter);
+            QStringList FileList;
```

Comment:
We don't use capital to start variables. I will rewrite your PR and submit it. Thank you for the catch.